### PR TITLE
Bug - 4679 - Fix Print Document Experiences

### LINKED ISSUE #4679: Experience not displayed properly on printed
## Where
- UAT server

## Description
The experience does not display properly when Printing a profile, Instead it shows a giant arrow.

## Screenshot
image

## Acceptance Criteria
- [ ]  we need to be able to print the content of the profile experience

   [image]: https://user-images.githubusercontent.com/305339/166570935-6425a615-82c7-4b2f-9840-5c7d698fa2ab.png
   [image-link]: https://example.com
   [figma]: https://go.talent.c4nada.ca/figma
   [sitemap]: https://go.talent.c4nada.ca/sitemap

### PR DESCRIPTION
**Pull request by esizer**
## 👋 Introduction

This fixes a class name change for the accordion icon which was allowing it to show up on printed user profiles. As well, it replaces the accordions with plain components when printing.

## 🧪 Testing

1. Build admin `npm run production --workspace=admin`
2. Navigate to a user profile
3. Click "Print profile"
4. Ensure the following:
    - chevrons for experience accordions are not present
    - content appears below the experience titles

## 🤖 Robot Stuff

Resolves #4679

### CODE REVIEW

File: frontend/admin/src/js/components/user/UserProfileDocument.tsx [hunk 1]
```diff
@@ -112,7 +112,7 @@ export const UserProfileDocument = React.forwardRef<
         .accordion-header-context .icon {
           width: 1.5rem;
         }
-        .accordion-header-wrapper svg {
```
Comment (mnigh) [inline frontend/admin/src/js/components/user/UserProfileDocument.tsx:115]: Would this instance of the selector need to be changed to `.Accordion__Chevron` as well?

https://github.com/GCTC-NTGC/gc-digital-talent/blob/1611e4794fbb981fb2639103a81da2a8bb8982a0/frontend/admin/src/js/components/poolCandidate/PoolCandidateDocument.tsx#L116-L118

Comment (esizer) [reply frontend/admin/src/js/components/user/UserProfileDocument.tsx:115]: Thanks! I forgot we had this in two places. Updated with [update accordion icon class name](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4750/commits/22c16d304553f193772d3a723cca7600f8beb39f)

**Review by mnigh — COMMENTED** [review]
This fixes the chevron, but the [linked issue](https://github.com/GCTC-NTGC/gc-digital-talent/issues/4679) mentions that the "experience does not display properly when Printing a profile". There are no acceptance criteria in the [issue](https://github.com/GCTC-NTGC/gc-digital-talent/issues/4679), so I'm not sure what the scope is, @Jerryescandon? If the original scope is only hidding the chevron on print styles, then I am happy to approve this PR and then can create another issue for the following:

1. accordion title and subtitle are not divided
2. accordion content is not rendered at all

## Screenshot
Screen Shot 2022-11-15 at 11 58 13

**Comment by Jerryescandon** [comment]
Almost, can we also include the expanded skill justifications? Styling is not super important, as long as we can read all info submitted. Experience info + skills and their explanations.

**Comment by esizer** [comment]
> Almost, can we also include the expanded skill justifications? Styling is not super important, as long as we can read all info submitted. Experience info + skills and their explanations.

Might need to expand this issue out a but an create a print only component for this. Our new accordions remove the content from the DOM when closed (default value) so there is no easy way to make the content appear when printing. 🤔

**Comment by Jerryescandon** [comment]
@gobyrne Should we close this one and add create a new issue for MLP?

**Review by petertgiles — APPROVED** [review]
Awesome! :sunglasses:

**Pull request merged**